salmon.md

```diff
@@ -0,0 +1,363 @@
+# Salmon
+
+## Utils
+
+These functions are used by the tests. Most utils are common code for setting up tests or asserting after the test is run.
+
+### `prepare_organism_indices`
+
+This function creates a base organism index and all of its relations which will be used by salmon in the job pipeline.
+
+### `prepare_job`
+
+This function creates all the prerequisites for the transcriptome index pipeline to be run.
+
+These prerequisites include:
+
+- ProcessorJob
+- Sample
+- OriginalFiles
+
+### `prepare_dotsra_job`
+
+This function is the same as prepare_job, but the Original File is sra.
+
+### `identical_checksum`
+
+Checks if two files have identical checksums.
+
+### `strong_quant_correlation`
+
+Checks if the two inputed quant files are strongly correlated.
+
+They are strongly correlated if both columns #3 and #4 (zero-indexed) of the two files have a correlation >= 0.99.
```

Comment:
This is a very useful level of detail, however it's a bit too far from the tests that actually use it.